### ISSUE TITLE
Ventoy fails to load OS on Macbook Pro 2012

### ISSUE THREAD

**Issue by enaran74** [ventoy/Ventoy#52]
While Ventoy will boot to the menu, as soon as you select any ISO file to mount and boot it fails with "Not a Secure Boot Platform 14"
It also flashes this same message up on initial boot but Ventoy seems to overcome the issue and loads to the menu.
Using 1.0.08 UEFI 
Tried latest ISO's of Ubuntudde-20.04 and Elementaryos-5.1 and Windows10_1909

**Comment by luzeagithub** [comment]
What are your settings here?
![image](https://user-images.githubusercontent.com/62355456/80950133-92588480-8df5-11ea-91ae-f2f32f7bec28.png)

**Comment by enaran74** [comment]
unfortunately the 2012 macbook pro's don't have access to this screen.
![macbookpro2012](https://user-images.githubusercontent.com/64768354/80951245-dcb02480-8e3a-11ea-9ac7-24a73760afaf.jpeg)

**Comment by chromer030** [comment]
@enaran74 You are still on old Mac OS X Lion , why you do not like to upgrade to Mojave or Catalina ?

**Comment by enaran74** [comment]
Unfortunately, when you load the recovery console this is all you get.
I was on Catalina prior to formatting and installing Linux.
Note. This is the recovery console provided by apple when there is no recovery partition on the disk.
It's loaded from the web.

**Comment by brandonkal** [comment]
The StartUp Security Utility is for configuring the T2 security chip mess. So it is only available starting with the 2018 MacBooks.

I have this same issue as well with Ventoy on my late 2013 MacBook Pro.

Is there anything I can do to help test this?

**Comment by xiaozhuai** [comment]
I use Macbook pro 2019. I can't boot to the menu. 

![82154740-74cdf500-98a2-11ea-9ca4-8b2dbc641c2a](https://user-images.githubusercontent.com/4773701/82518277-999bc400-9b51-11ea-89a4-58c4527c5415.jpeg)

**Comment by AlexOwen** [comment]
I also have the same issue on a mid-2014 MacBook Pro with Mojave. The same "No ER state object" error as @xiaozhuai above.

**Comment by mikanesth** [comment]
> I use Macbook pro 2019. I can't boot to the menu.
> 
> ![82154740-74cdf500-98a2-11ea-9ca4-8b2dbc641c2a](https://user-images.githubusercontent.com/4773701/82518277-999bc400-9b51-11ea-89a4-58c4527c5415.jpeg)

same for me here

**Comment by NewtonGimmick** [comment]
This is the only result when I search this error. It also happens that I am attempting to use Ventoy as well.  I am not sure where the disconnect is, but I get the same error on my Macbook 11,1 which does not have T2 so you can't access the menu others here have spoken about.

**Comment by brandonkal** [comment]
Ventoy just doesn't work on Apple hardware. I've switched to easy2boot which works well (though I am not using it to boot hfs installers).

**Comment by peterdk** [comment]
Looked at Ventoy to see if it could replace my Zalman VE-350, that can boot any IOS you throw at it. Ubuntu goes ok, but my collection of OSX ISO's give me the same 'No ER state object [..]'. Too bad. Hope it will be made possible in the future.

**Comment by julianpoemp** [comment]
same issue like the other people before. It's sad, that ventoy is not working on Macbooks 😞 
![IMG_20201017_102648](https://user-images.githubusercontent.com/25926588/96331457-a586a600-105d-11eb-9a34-9c7c010f2313.jpg)


My System:
MacBook Pro (Retina, 15', Mitte 2014)
Intel Iris Pro 1536 MB
MacOS Catalina 10.15.6 (19G2021)

**Comment by julianpoemp** [comment]
@ventoy is there any progress on this issue? When I try booting on a Mac with a USB stick created by another software (e.g. Rufus) it works all fine...

**Comment by skulldealer** [comment]
Please Don't expect anything to happen asap even I did not get any response on an similar issue over a week ago. I am particularly not able to boot 2 iso from ventoy. So please wait for the next release when all our issues will be addressed and solved.

**Comment by julianpoemp** [comment]
@skulldealer this issue is much older than just one week. Of course I don't expect issues getting solved asap, but 5 months is a time we may ask about any news...

**Comment by skulldealer** [comment]
As per my knowledge ventoy developer has a very limited development environment. Ventoy has a ThinkPad I have seen his videos. I don't think he has a macbook pro specifically to test ventoy.

**Comment by skulldealer** [comment]
> @skulldealer this issue is much older than just one week. Of course I don't expect issues getting solved asap, but 5 months is a time we may ask about any news...

I did not even get a single reply regarding my issue

**Comment by skulldealer** [comment]
You could try yumi tool 
it needs windows to install but works with Mac's . Could use a vm

**Comment by julianpoemp** [comment]
@skulldealer 

1. I know other tools, but no alternative is as easy as ventoy.
2. Why do you make so many comments? Please edit your latest comment instead.
3. What are you talking about? You created 5 issues here and 4 were answered by ventoy. The latest one is not answered for now, **but it's just 9 hours old**???

**Comment by skulldealer** [comment]
I reopened the latest one again after closing the one I asked a week ago coz it was dead

**Comment by skulldealer** [comment]
I want knoppix iso support on ventoy great tool for recovery of damaged files from hardisk

**Comment by skulldealer** [comment]
Was polite enough to provide the direct download links for the iso files with the issue.

**Comment by skulldealer** [comment]
By the way there is another active forum on official ventoy site. You could go there maybe

**Comment by skulldealer** [comment]
You should probably close this one and open a same new issue on GitHub

**Comment by skulldealer** [comment]
I have an idea

Install ventoy on usb with setting on for preserve some space at end. the iso files you are able to boot keep in ventoy partition and use unetbootin to flash the iso u can't boot from ventoy to the non ventoy partition that could help.

**Comment by skulldealer** [comment]
> I have an idea
> 
> Install ventoy on usb with setting on for preserve some space at end. the iso files you are able to boot keep in ventoy partition and use unetbootin to flash the iso u can't boot from ventoy to the non ventoy partition that could help. 

May create 2 bootbale usb in one

**Comment by skulldealer** [comment]
https://unetbootin.github.io/

It is available for macos

**Comment by AlexOwen** [comment]
@skulldealer please stop posting. Every message you post is emailed to 10+ other people.

This is not a forum for discussion, it is an issue tracker. If you have something constructive, post it, if not, post somewhere else.

**Comment by skulldealer** [comment]
Ok sry

**Comment by skulldealer** [comment]
issue resolved!!!!!!!!

Mac os can now work with ventoy 100 percent done I did it 

Need to integrate unetbootin with ventoy.

Please give me some time 


I assure you that's possible

Now ventoy can work with macos with a plugin possibly. Please wait as I test other iso. So far I got success with ubuntudde.iso latest


Pls wait I am sure there is a way to solve this issue . after I do my research over this I will need forward my research to ventoy which will be released later.

**Comment by skulldealer** [comment]
Need to change partition structure for mac support. Possibly there are 2 uefi partions required

Such usb will also be compatible with other os like windows laptop!

**Comment by julianpoemp** [comment]
@skulldealer I try to say it as friendly I can: STOP POSTING MORE COMMENTS!! IF YOU HAVE ANY NEWS EDIT YOUR LATEST POST, THANKS!

I know, you just want to help, but please please stop posting so many comments. This is not a chat!

**Comment by xiaozhuai** [comment]
@skulldealer Please! Update your answer than post a new one! I've received many email.

**Comment by skulldealer** [comment]
Final view on topic-

I will be creating a new issue with same topic and restart fresh. I will be uploading my results and research on the topic.I need to confirm my reaserch. I need volunteers. Requirements are that the volunteer should have both a macbook and a windows device. I will be asking the volunteers to persorm some simple tasks on ventoy. Voulteers need not spend a lot of time. I will be uploading simple steps and voulteers need to tell me if they are working or not. As a reply I want to know if you will be a volunteer or not. Also please list your favourite iso files u wish to boot. The most common of them will be tested.





What I have found so far?

Ok now listen there are two tools ventoy and unetbootin. Both on GitHub. So far u should know simply that using both together could simply be the solution. I need to work intensively see haw good of a combination it will be.


In short I want you to trust me, tell me your most used iso files with mac and plz volenteers intrested please let me know. Requirements to be volenteers 8s to have both windows and mac.

**Comment by skulldealer** [comment]
Mind blowing breakthrough achieved!!!!!!

Rejoice my friends as the new mac suppor mode has got. More than what meets the eye.

It looks like this project gives ventoy 100 percent success rate. Now using this so called mac. Mode we can boot any iso on any device mac or. Windows.

The results so far are too good.

I can just create a roadmap or a blueprint of what all needs to be done to achieve this. The real execution and coding has to be done by ventoy itself.

I literally cannot believe what I have figured.

Will combine all my results and findings in one post and post it in a new thread as well as on this one. Initially this will be completely experimental but with some time put into it it will achieve great stability and soon there will be no unsupported iso for ventoy.


Reminder - before showcasing my findings I need some volunteers to again confirm the same findings on their side will be very simple. Pls voulteer if possible. Requirements are-
1 a mac real physical machine not vm
2 a windows device real physical machine not vm
3 a usb to spare 


Please support this by voulnteering and telling me your favourite iso which you would like to run on Mac's. Will test those as well. Currently testing out the most common of them.

Thank you.

**Comment by skulldealer** [comment]
important update - 

after intensive testing,
as of now 99 percent of all mainstream debian and ubuntu based distro are working.
as of now arch is not working.

here is a list of all the iso files i tried and which all worked

elementaryos-5.1-stable.20200814.iso
gparted-live-1.1.0-5-amd64.iso
kali-linux-2020.3-live-amd64.isohkubuntu-20.04.1-desktop-amd64.iso
linuxmint-20-cinnamon-64bit.iso
MX-19.2_September_x64.iso
neon-user-20201015-0947.iso
Parrot-security-4.10_amd64.iso
tails-amd64-4.10.iso.
ubuntu-20.04.1-desktop-amd64.iso
ubuntu-budgie-20.04.1-desktop-amd64.iso
ubuntudde-20.04.1-amd64-desktop.iso
Windows.iso
xubuntu-20.04.1-desktop-amd64.iso
Zorin-OS-15.3-Core-64-bit.iso
Zorin-OS-15.3-Lite-64-bit.iso







i want volunteers with a both mac and a windows pc as well to confirm the same results.
u will have to test mint, ubuntu and elementary os only. will be very simple.

once the mac support is enabled the issue of unsupported iso can be solved later.

after confirming with the volunteers i will create a final verdict comment on both this thread and a new one.
it will have all the info to help ventoy developer enable macos support and suggestions for ventoy v2.0
the first release will be not the final, support for many other iso can be added afterwards for mac users.



i want volunteers to opt in fast to speedtrack macos support deployment for ventoy.

**Comment by skulldealer** [comment]
other than mainstream debian and ubuntu distro, mx linux and other and other anti-x iso will be supported after the first release. also windows 10 2004 32 bit and 64 bit will also be bootable from mac just in case.

updates regarding unsupported iso will be deployed slowly as first unetbootin has to update itself and then ventoy.
could be possible that all the plugins wont be available. please wait for updates as they will be ported for macbooks.

**Comment by skulldealer** [comment]
Any voulteer?

**Comment by julianpoemp** [comment]
@skulldealer why don't you summarize your explorations to one comment only like enyone else? Why do you ignore that we don't want to get 20 emails because of your comments?

Does your solution still needs unebootin or other additional software? If so it's not a solution, it's a workaround. I think the most of us want a solution implemented in ventoy without using any further software.

**Comment by skulldealer** [comment]
The fundamentals of unetbootin can be applied into ventoy to achieve macbook support.

**Comment by xiaozhuai** [comment]
OK! I am going to unsubscribe this

**Comment by skulldealer** [comment]
Please contribute by volunteering. That's the least I can say😔

**Comment by captainbrown** [comment]
> Please contribute by volunteering. That's the least I can say😔

I can do some testing.  I have a Desktop running windows and a 2017 Macbook Air.  Trying to get mint running and it gives me a blank screen and a frozen cursor. Do you have a build for me to test?

**Comment by WestCope** [comment]
I'm up to volunteer. Have both systems and multiple machines to test on!

**Comment by skulldealer** [comment]
@captainbrown , @WestCope 
please check the new thread i have opened for macbook support and voulnteers.
thank you.

**Comment by skulldealer** [comment]
I have shared the post in new thread with instructions for volunteers. In case if you want more support please let me know. I will post another step by step instruction guide for you all. You have to folplow the steps and post your observations.

**Comment by bartatgithub** [comment]
Hi 

Tested the latest Ventoy 1.0.29 with different Linux Distro's on different Apple computers and this are the results: 


MacBook Pro 2015 Catalina SIP enabled = Doesn't work "Not a Secure Boot Platform 14"
MacBook Pro 2015 Catalina SIP disabled = Doesn't work "Not a Secure Boot Platform 14"
iMac 2013 Catalina SIP enabled = Works! 
iMac 2019 Catalina SIP enabled = Works! 

The problem is that I need it for the MacBook Pro and not for the iMac's. :-( 
I have tested all with secure boot support off and on. 

How can I get this working on the MacBook Pro 2015? 

Thanks for this amazing tool!

**Comment by julianpoemp** [comment]
@bartatgithub it will not work as long as @ventoy does not fix it.

With ventoy application you could preserve space on your USB stick and install a linux system on that partition next to ventoy. For this I used unebootin. Perhaps it is possible to install another multi-boot system on that partation, but I don't know if it works.

## Workaround: Installation of Kali next to ventoy partitions

I did that with Kali on my USB stick to have a system with persistence:

1. Install ventoy and do the option to preserve data at the end of the usb stick.
2. Use a disk partition tool and split the last partition of the USB stick into two pieces: 1) Partition for kali system FAT32, 2) Partition ext4 with label "persistence".
3. Use unebootin to write the ISO to the first partition (third partition over all).
4. On the second partition (fourth partition over all) create a file called "persistence.conf" and add content "/ union".

Unfortunately my solution is still a workaround. I hope ventoy fixes this problem so that no workaround with third party software is needed.

**Comment by skarchmit** [comment]
> same issue like the other people before. It's sad, that ventoy is not working on Macbooks 😞
> ![IMG_20201017_102648](https://user-images.githubusercontent.com/25926588/96331457-a586a600-105d-11eb-9a34-9c7c010f2313.jpg)
> 
> My System:
> MacBook Pro (Retina, 15', Mitte 2014)
> Intel Iris Pro 1536 MB
> MacOS Catalina 10.15.6 (19G2021)

It used to work very well for me after i already installed ubuntu on another partition... few updates in ubuntu itself, this screen started popping up. And then i was no longer able to boot from ventoy

**Comment by bartatgithub** [comment]
Yes, it's strange; Not working on a MacBook Pro but working on a iMac with same system and on a MacMini with older system. 


Question to check because I start to think that could be the problem: 
Do you have firmware (hardware) password on? 

I can't check because have that on and can't put it off since the latest firmware update. 



On 22 January 2021 09:01:15 GMT+01:00, Serge Karchmit <notifications@github.com> wrote:
>> same issue like the other people before. It's sad, that ventoy is not
>working on Macbooks 😞>
>>
>![IMG_20201017_102648](https://user-images.githubusercontent.com/25926588/96331457-a586a600-105d-11eb-9a34-9c7c010f2313.jpg)>
>> >
>> My System:>
>> MacBook Pro (Retina, 15', Mitte 2014)>
>> Intel Iris Pro 1536 MB>
>> MacOS Catalina 10.15.6 (19G2021)>
>>
>It used to work very well for me after i already installed ubuntu on
>another partition... few updates in ubuntu itself, this screen started
>popping up. And then i was no longer able to boot from ventoy>
>>
>-- >
>You are receiving this because you were mentioned.>
>Reply to this email directly or view it on GitHub:>
>https://github.com/ventoy/Ventoy/issues/52#issuecomment-765215830

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by damienbutt** [comment]
Is there any update on a solution for this?

I have just setup Ventoy to find that it will not work with my Macbook Pro. It's boots to the boot menu and I can pick an OS and it will boot that OS, but once it's booted nothing works. No mouse, no keyboard, no trackpad etc.

**Comment by kkovacs** [comment]
Something that might help this in a direction: I'm having the same issue with Ventoy as everybody else here (Mac, same screen). It doesn't boot any Linux **except for [Tails](https://tails.boum.org/)**. **It does boot Tails.** (Tails is an animal, it boots on everything.)

If we could figure out what Tails does differently, then maybe we could reproduce it using the _inject_ plugin or something?

Any thoughts?

**Comment by SomeUserName1** [comment]
Same issue here, 2015 MacBook Pro.
Tried Arch, Ubuntu, and Fedora.

**Comment by F9Alejandro** [comment]
> unfortunately the 2012 macbook pro's don't have access to this screen.
> ![macbookpro2012](https://user-images.githubusercontent.com/64768354/80951245-dcb02480-8e3a-11ea-9ac7-24a73760afaf.jpeg)

Technically you do it is the firmware password. However it is an old version of the UEFI firmware that needs to be updated due to not having secure boot

**Comment by Diacius** [comment]
On my Macbook Pro 2011 (no firmware password etc.), the menu loads fine but as soon as I select an ISO it goes to a blank screen with one `_` if the same ISOs are flashed to a drive with etcher,rufus etc the ISO boots fine.
If I enable `memdisk` and select an ISO, the message `Loading ISO into memory` stays on screen forever.
This is really annoying as my all in one 128GB USB that I have ventoy and some utilities on is totally useless if I need to run linux on a mac. @ventoy is currently on break and I know he doesn't have a mac, but I'd be happy to test dev builds etc.

**Comment by Diacius** [comment]
>Something that might help this in a direction: I'm having the same issue with Ventoy as everybody else here (Mac, same screen). >It doesn't boot any Linux except for Tails. It does boot Tails. (Tails is an animal, it boots on everything.)
>If we could figure out what Tails does differently, then maybe we could reproduce it using the inject plugin or something
>Any thoughts?

**I'll try tails when I get the chance and see if it boots**, maybe it would also help to see what Tails does to "boot on everything" to get other ISOs to boot on mac

**Comment by steve6375** [comment]
Does Easy2Boot work?

**Comment by Diacius** [comment]
> Does Easy2Boot+agFM work? agFM also uses grub2.

I've never tried either of the things you mentioned. However, if easy2boot uses the same method as Ventoy (IDK) surely the same problem would exist?
I might try it as well, however **this is still a workaround, and not a solution** I would still prefer that **_Ventoy worked_** easy2boot has a pretty bad website and seems to be less popular. Also could you elaborate on what the point of agFM is? I didn't really follow the info on the website...

**Comment by steve6375** [comment]
see www.easy2boot.xyz
It contains 3 menu systems, grub4dos E2B (for legacy), agFM grub2 (legacy and UEFI) and Ventoy (currently old version).
Since agFM uses grub2 - if it works then it indicates a problem peculiar to Ventoy.
Another avenue is to use the Ventoy F6 extended menu feature and write a grub2 .cfg file to boot a Linux ISO and see if that works.
At least more knowledge will be obtained.

**Comment by Diacius** [comment]
I would prefer to use plain Ventoy as it has plenty of good features, and I understand how it works. But I could compromise with easy2boot+Ventoy / easy2boot+agFM. Although the HFS boot feature seems tempting. Also agFM seems pretty nice. I will try this and report back on what happens. As you said 'At least more knowledge will be obtained.'

**Comment by Diacius** [comment]
> > unfortunately the 2012 macbook pro's don't have access to this screen.
> > ![macbookpro2012](https://user-images.githubusercontent.com/64768354/80951245-dcb02480-8e3a-11ea-9ac7-24a73760afaf.jpeg)
> 
> Technically you do it is the firmware password. However it is an old version of the UEFI firmware that needs to be updated due to not having secure boot

Does it get updated during MacOS updates? I assume it does as I currently dual boot Fedora+OS X El Capitan (I had to manually flash Fedora) so I don't know if that version of Mac OS Recovery has secure boot. I get a firmware password option but thats it.

**Comment by Diacius** [comment]
If someone can test for @ventoy and try the stuff mentioned here (I might) that would help.
I created #954 but closed it because it was a duplicate. Maybe we need to consolidate all the MacOS issues like #35 #871 #680 #917 #686 here or in a new one. Because @ventoy doesn't have a mac this makes ths difficult, maybe someone could find the changes needed?

**Comment by Diacius** [comment]
**#7 states **
> NOTE : Any "Install macOS ..." created bootable .dmg's (which are just .img files) , ISO or RAW , don't currently boot on Macs (through ventoy) due to how their bootloaders are setup compared to normal UEFI bootloaders.

So someone needs to work out the differences and see if there can be a Mac mode or something...

**Comment by bartatgithub** [comment]
Wow, this discussion is going on. 
I imagin that all people in this discussion can and are willing to test. 
But does Ventoy has time and energie to work on this? 
So, i am willing to test but first i have to fix the 'locked out firmware password' problem that i have. 


Years ago i have put a 'firmware password', with a symbol in it, because that's a nice extra securety but with one of the latest Catalina Security Updates the 'firmware' was updated. Since then when i try to put 'firmware password' off to test Ventoy it says that i can't use that symbol in a 'firmware password'. So, i can't put it off or change it anymore. Searched the web but no fix for that one. If somebody can help with that i can test some of the possibilities out here. 

p.s.: Meanwhile i am thinking to leave my MacBook Pro, it's from 2015 and go for a Notebook, it's cheaper for new hardware plus easier to install Linux on it. ;-)

**Comment by Diacius** [comment]
> Wow, this discussion is going on.
> I imagin that all people in this discussion can and are willing to test.
> But does Ventoy has time and energie to work on this?
> So, i am willing to test but first i have to fix the 'locked out firmware password' problem that i have.
> 
> Years ago i have put a 'firmware password', with a symbol in it, because that's a nice extra securety but with one of the latest Catalina Security Updates the 'firmware' was updated. Since then when i try to put 'firmware password' off to test Ventoy it says that i can't use that symbol in a 'firmware password'. So, i can't put it off or change it anymore. Searched the web but no fix for that one. If somebody can help with that i can test some of the possibilities out here. The weird thing is that i still can enter the 'firmware password' and that works, for example to enter 'Recovery Mode' but i can't put it off or change it.
> 
> p.s.: Meanwhile i am thinking to leave my MacBook Pro, it's from 2015 and go for a Notebook, it's cheaper for new hardware plus easier to install Linux on it. ;-)

Sadly, AFAIK you can't downgrade mac firmware, I know because I installed snow leopard onto a mac with El Capitan era firmware. (long story basically i cant use "Install mac os" so i had to upgrade from Snow Leopard as its the last to come on DVD)

Yeah, the only reason I use a mac is because it's the only non-chromebook laptop I have, I have a PC but its handy to have a laptop to use around the house etc. I would buy a linux laptop, but I'm still in school so I can't afford one as they tend to be more expensive than parts for a pc. Apparently as read in #7 , macs cant boot Ventoy because of their proprietary EFI (Classic Apple lol).

**Comment by bartatgithub** [comment]
Yes, i know! So, for me difficult to test. 
I have just checked and Ventoy himself has never replied on this post and i read somewhere that he doesn't work for Mac computers. So, first we need some answere from him i think.

**Comment by Diacius** [comment]
> Yes, i know! So, for me difficult to test.
> I have just checked and Ventoy himself has never replied on this post and i read somewhere that he doesn't work for Mac computers. So, first we need some answere from him i think.

Yes, he doesn't have a mac.
Also, I don't mean to be rude (i noticed your from belgium :) ) but check out deepl.com for translation, its way better than google translate

**Comment by bartatgithub** [comment]
... i know and use sometimes deepl, G-free i am ;-), but I prefer to translate simple replies myself and not to depend on 'translators'. But that is another story.

**Comment by lerow12** [comment]
Just tried booting from ventoy on my Macbook Pro and now I can't even book MacOS. It just sits with the screen off. Tried to hard restart it but nothing happens at all. Any suggestions?

**Comment by Diacius** [comment]
> Something that might help this in a direction: I'm having the same issue with Ventoy as everybody else here (Mac, same screen). It doesn't boot any Linux **except for [Tails](https://tails.boum.org/)**. **It does boot Tails.** (Tails is an animal, it boots on everything.)
> 
> If we could figure out what Tails does differently, then maybe we could reproduce it using the _inject_ plugin or something?
> 
> Any thoughts?

Intresting. My mac boots most linux if you flash it to a drive using Rufus or Etcher or whatever. I need to get round to trying Tails, perhaps with ventoy?

What's especially odd about this whole issue is that the menu appears fine, but Ventoy can't load the OS. But if you use a distro iso flashed to usb it works fine! What's the difference between the two?

**Comment by DarkLegacy248** [comment]
Has there been any progress on this issue?

**Comment by theAeon** [comment]
Has anyone tried this with the EFI partition as HFS+ instead of FAT32? I have to imagine that'd help matters.

**Comment by lsnch** [comment]
So the fact that only Tails is working got me curious and I did some digging. I'm on MBA2013.

Boot happens only from **tails.img** (for USB sticks), but not from **tails.iso** (for DVD). Iso (both tails and all other distros) can only be mounted as cd9660, whereas img (from tails) is a GPT partitioned image with entire payload (1G+) on its single EFI partition (the special one, with partition type ef00):

![Screenshot 2022-02-04 at 06 01 29](https://user-images.githubusercontent.com/20382674/152465269-71171f5a-8509-4a31-82f7-e802deed23af.png)

I tried making tails.iso work. When loading a drive, BIOS systems look for code on the first sector (MBR, 512 bytes), ahead of all partitions, which is where the first stage of grub would normally live. AFAIK, UEFI+GPT boot mechanism (like tails.img, according to me) has no such need. There, MBR has no code, and all loaders live on the special EFI partition (which typically should be 200M). So, I thought if I just extracted tails.iso onto the EFI inside gpt image it would work.

Perhaps ventoy unpacks those images differently and reads MBR, somehow? It could load itself as EFI, but then execute images as MBR? Maybe tails.img contains boot code, in the first sector? Or maybe grub loads and silently fails? I'm out of ideas, maybe I'm missing something.


![Screenshot 2022-02-04 at 03 25 12](https://user-images.githubusercontent.com/20382674/152468263-91c81153-f64e-4aff-90e4-07d7b160dadb.png)
![Screenshot 2022-02-04 at 03 34 18](https://user-images.githubusercontent.com/20382674/152468266-e3db3110-90ad-441d-a718-6088e012d08f.png)

**Comment by Diacius** [comment]
That's, really weird! For me grub loads and displays a menu, and all of the Ventoy functions seem to work. But if i select any ISO, with or without the memdisk option. I just get an endless black screen

**Comment by blindelectron** [comment]
I have the same issue on MacBook Pro 2015, and ventoy loads fine, but as soon as I try to book any operating systems, the screen goes black for a minute, then it comes up with a message, not a secure platform 14. Unless of course, I load either, tails.IMG, or cloudready.IMG

**Comment by betabot7** [comment]
For me, on a 2012 Mac mini, the latest version of Ventoy showed up both as EFI and Windows.  EFI gives the errors mentioned above, but when I selected windows, I was able to boot a proxmox 7.2 iso.  Note, I did not have windows installed in bootcamp.

**Comment by manuelv04** [comment]
no solution yet??

**Comment by Lyapsus** [comment]
Also Arch iso is able to boot in grub2 mode

**Comment by qral** [comment]
Tried latest Ventoy (1.0.87) on Win, removed _Secure boot support_  and booted live distro (Kali and Ubuntu) via grub2 mode on the macbook mid-2015 fine.
Also if I tried normal mode I had to reset SMC afterwards, because of black screen boot 😠

**Comment by remym19** [comment]
I could also overcome the issue by booting the ISO in Grub2 mode from Ventoy.

**Comment by jrussell88** [comment]
I have a mid-2014 Intel-based Macbook Pro (11,2), running Refind, Ubuntu and OSx El Capitan. SIP is disabled, and it predates the T2 chip. The hard drive is GPT. I re-enabled SIP without any effect on the error: "Not a Secure Boot Platform 14" 

I tried grub.efi and grub64_real.efi. 

From grub.efi, Ventoy 1.0.88 brings up the Grub menu with the two available Ubuntu ISOs, then offers a choice of boot methods (normal mode, grub2 etc). Normal mode hangs with the "Not a Secure Boot Platform 14" error whilst booting Ubuntu 22.04.2.

From the grub.efi, Grub2 mode boots successfully.

**Comment by super-15** [comment]
just... dont use ventoy for it...?

**Comment by exee21** [comment]
And here it is 2023. --- I just used latest ver of Ventoy with ubuntu-desktop.iso and booted it with grub2 with no problems at all. -- Resolved.

**Comment by iwannet** [comment]
Not resolved, macOs images still dont boot using ventoy

**Comment by samvv** [comment]
Grub2 works here too on an old MacBook 2014 model, for Ubuntu 24.04

**Comment by kkovacs** [comment]
I can confirm that grub2 is working for me with Lubuntu.

Probably the ticket can be closed, @ventoy ?

Thanks for the great work you are doing on this project!

**Comment by pierredGitHub** [comment]
Absolutely agree. We need macOS ISOs to boot.

**Comment by kkovacs** [comment]
Oh, the macOS ISO was not on my radar, only the Linux-es. Okay. :)

**Comment by iplayradionetworks** [comment]
How did you get on this menu ?

> > What are your settings here? ![image](https://user-images.githubusercontent.com/62355456/80950133-92588480-8df5-11ea-91ae-f2f32f7bec28.png)
> 
> Restricted boot is only on T2 Macs.

**Comment by brainrom** [comment]
Tested on Mac Mini Late 2012. Ventoy version 1.1.05. Mac EFI Version 429
Firmware password doesn't affect the issue, still `Not a secure platform 14` on Ventoy start for a second, and the same error when trying to boot images in normal mode.
But successfully booted Ubuntu 24.04 and Arch 2025.07 in GRUB2 mode. 
Tails iso works absolutely different: no normal/grub2/mem mode menu, just going directly to grub2 menu (different from normal grub2 menu). But fails with error `Unable to find a medium containing a live system` and stucks in initram.

**Comment by ColleagueRiley** [comment]
Seems to be an issue on my 2012 x86_64 imac as well. 

Doesn't work with Linux Mint (XFCE) and SystemRescue, but it works fine with Ubuntu 24.04.

**Comment by splitsnapdonkee** [comment]
October 2025. MacBook Air early 2015: “Not a secure boot platform 14.” Then I pressed the off button and the laptop never started up again. Wish I’d seen this thread before lol. Kinda wondering what could have bricked it though in all honesty it was barely hanging on to begin with

**Comment by Jukelyn** [comment]
I use ventoy to install different linux OS onto my 2018 mac minis (macmini8,1) that I use as servers. Do you guys not see the EFI boot option when holding the option key down when booting? As for macOS, I found that creating different partitions and using the createinstallmedia command through whichever OS pkg to work for that.

This is a guide I wrote that goes over the steps I take every time: https://jukelyn.com/posts/ubuntu
